tests/util.py

```diff
@@ -117,7 +117,8 @@ def mkfs(device, fs="ext4"):
     if (fs == "xfs"):
         cmd = ["mkfs.xfs", device, "-f"]
     else:
-        cmd = ["mkfs." + fs, "-F", device]
+        # Disable lazy init to facilitate that no additional IO will take place during tests
+        cmd = ["mkfs." + fs, "-F", "-E", "lazy_itable_init=0,lazy_journal_init=0", device]
```

Comment:
I bet, in most cases, user's envs have no this parameter. Please explain why it makes sense to do in tests despite the fact it's not done in real driver usage conditions?

Reply:
Because if we want to fill the cow file [almost] completely in `test_modify_origin`, we are not interested in the potential tracking of lazy init write operations after `mkfs`.